packages/client/src/containers/RetroPage/RetroPage.jsx

```diff
@@ -0,0 +1,93 @@
+
+import React, { useEffect, useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { apiURL } from "../../apiURL";
+
+export default function RetroPage() {
+  const navigate = useNavigate();
+  const [currentDate, setCurrentDate] = useState("");
+
+const updateButton = () => {
+    navigate("/updateTeamPage");
+  }
+
+
+  const startRetro = () => {
+    // Generate retro code and navigate to RetroPage2
+    const retroCode = Math.random().toString(36).substring(2, 10);
+    navigate(`/retroPage2/${retroCode}`);
+  };
+
+
+  const pastRetro = () => {
+    navigate("/retroHistory");
+  };
+
+  const submit = async () => {
```

Comment:
You should think about choosing a convention for naming handler functions. For example, `handleSubmit` or `onSubmit`.